backend/windmill-api/src/resources.rs

```diff
@@ -322,8 +331,22 @@ async fn get_resource_value_interpolated(
 
     let value = not_found_if_none(value_o, "Resource", path)?;
     if let Some(value) = value {
+        let job = if let Some(job_id) = job_info.job_id {
+            let mut tx = user_db.clone().begin(&authed).await?;
+            let result = sqlx::query_as::<_, QueuedJob>(
+                "SELECT * FROM queue WHERE id = $1 AND workspace_id = $2",
+            )
+            .bind(job_id)
+            .bind(&w_id)
+            .fetch_optional(&mut *tx)
+            .await?;
+            tx.commit().await?;
```

Comment:
you do not need to load the full job (which may contain some info like the raw_flow !